src/examples.md

```diff
@@ -1510,6 +1511,39 @@ moving to the miniapps.**_
 <br></div>
 
 
+<div id="lorentz" markdown="1">
+## Lorentz Miniapp: Particle Trajectory Calculation
+<img class="floatright" src="../img/examples/lorentz.png">
+
+This miniapp reads VisItDataCollection objects produced by applications such as
+Volta and Tesla (described above) and computes particle trajectories subject
+to the electric and magnetic fields described therein.
+
+This code uses the Boris algorithm to approximate the Lorentz force,
+
+$$\frac{d\vec{p}}{dt} = q \left(\vec{E} + \vec{v}\times\vec{B}\right),$$
+
+exerted on a test particle with a given mass, charge, initial position and
+momentum.
+
+The Lorentz miniapp musts be run on the same number of processors as the
+applications which produced the VisItDataCollection objects. However, the
+electric and magnetic fields do not need to be defined on the same mesh. The
+Lorentz miniapp relies on the `FindPoints` functionality available through
+MFEM's parallel mesh classes to query the field values even when those field
+values are located on different processors.
+
+For more details, please see the [documentation](electromagnetics.md) in the
+`miniapps/electromagnetics` directory.
+
+_The miniapp has only a parallel
+([lorentz.cpp](https://github.com/mfem/mfem/blob/master/miniapps/electromagnetics/lorentz.cpp)) version.
+**We recommend that new users start with the example codes before
```

Comment:
It would be good to mention particular examples to review that will help a user understand this miniapp.

Reply:
Done